src/alpaca_eval/decoders/jinachat.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import multiprocessing
+from functools import partial
+from typing import Sequence
+import requests
+import json
+import os
+import time
+from .openai import _prompt_to_chatml
+from .. import utils
+
+__all__ = ["jina_chat_completions"]
+
+
+def jina_chat_completions(
+    prompts: Sequence[str],
+) -> dict[str, list]:
+    """Get jina chat completions for the given prompts. Allows additional parameters such as tokens to avoid and
+    tokens to favor.
+
+    Parameters
+    ----------
+    prompts : list of str
+        Prompts to get completions for.
+    """
+    n_examples = len(prompts)
+    api_key = os.environ.get('JINA_CHAT_API_KEY')
+
+    if n_examples == 0:
+        logging.info("No samples to annotate.")
+        return {}
+    else:
+        logging.info(f"Using `jina_chat_completions` on {n_examples} prompts.")
+
+    prompts = [_prompt_to_chatml(prompt.strip()) for prompt in prompts]
+    num_processes = min(multiprocessing.cpu_count(), 4)
```

Comment:
maybe make the 4 a parameter to the function, like for [anthropic](https://github.com/tatsu-lab/alpaca_eval/blob/40282f5149953f175992c30b0eb6406372fb62fa/src/alpaca_eval/decoders/anthropic.py#L18) or openai